Return Elements Using CSS Selectors/app.js

```diff
@@ -0,0 +1,7 @@
+let navigationLinks = document.querySelectorAll('nav a');
+let galleryLinks = document.querySelectorAll('#gallery a');
+let footerImages = document.querySelectorAll('footer img');
```

Comment:
Please use `const` since the value was not reassigned.

Reply:
Thanks, @lycoangelo, this is now updated.